README.md

```diff
@@ -117,6 +114,43 @@ Notice that values are converted to pence for downstream processing and the invo
 }
 ```
 
+### Example response message
+
+An acknowledgement response message is published after enrichment of every payment request to be optionally consumed by source systems.
+This message includes the enriched payment request and whether not the service accepted the request.
+
+{
+  "paymentRequest": {
```

Comment:
`originalSettlementDate` shouldn't be there in the first place.  I've now removed all trace of it.

It's automatically determined in `ffc-pay-processing` from the return file data.  No way it can be known at this point.